src/cljs/hatti/views/dataview.cljs

```diff
@@ -27,7 +30,7 @@
            :label "Summary Charts"
            :component chart-page}
    :details {:view :details
-             :label "Details"
+             :label "Settings"
```

Comment:
This no longer reflects the current state of things. `:component` value should probably be `settings-page`.